_apps/pull.md

```diff
@@ -25,31 +26,33 @@ organizations:
 - tiaanduplessis
 ---
 
-Keep your forks up-to-date via automatic pull requests, merges, or hard resets.
+[![Probot #1 Most Installed](https://badgen.net/badge/probot/%231%20most%20installed/orange?icon=dependabot&style=flat&cache=86400)](https://probot.github.io)
```

Comment:
Could you remove this please, it's not true, I'm afraid. WIP has 7356 right now, and Stale has far, far more. I didn't check the other apps